spec.md

```diff
@@ -0,0 +1,442 @@
+consist of one JSON object that contains both context and data).
+
+### namespace
+* Type: String
+* Description: Identifier that uniquely identifies the organization publishing
+  the event.
+* Examples:
+  * kafka.apache.org
+  * com.microsoft.azure
+
+### event-type
+* Type: String
+* Description: Type of the event.  Producers can specify the format of this,
+  depending on their service.  This enables the interpretation of data, and
+  can be used for routing, policy and more.
+* Examples:
+  * customer.created
+* Constraints:
+  * Required
+* Notes:
+  * It is up for discussion whether namespace could be included as a prefix in
+    event-type.
+  * It is up for discussion whether the event-type should also include a
+    version.  It’s currently included in a separate attribute titled
+    event-type-version.
+  * It is up for discussion whether this specification should enforce a type
+    format.
+
+### event-type-version
+* Type: String
+* Description: The version of the event-type.  This enables the interpretation
+  of data by eventual consumers, requires the consumer to be knowledgeable about
+  the producer.
+* Notes:
+  * It is up for discussion whether this is applicable to the whole event or
+    to the data payload alone.  Currently, event-type-version covers the data,
+    while open-events-version covers the context.
+
+### open-events-version
+* Type: String
+* Description: The version of the OpenEventing specification which the event
+  uses.  This enables the interpretation of the context.
+
+### resource
+* Type: Object
+* Description: This describes the software instance that emits the event at
+  runtime (i.e. the producer).  It contains sub-properties (listed below)
+
+
+### resource-type
+* Type: String
+* Description: Type of the event source. Providers define list of event sources.
+* Constraints:
+  * Required
+* Examples:
+  * s3
+
+### resource-id
+* Type: String
+* Description: ID of the event source.
+* Constraints:
+  * Required
+*Examples:
+  *my.s3.bucket
+
+### event-id
+* Type: String
+* Description: ID of the event.  Can be specified by the producer.  The
+  semantics of this string are explicitly undefined to ease the implementation
+  of producers.  Enables deduplication.
+* Examples:
+  * A database commit ID
+* Constraints:
+  * Required
+  * Unique per producer
+
+### correlation-id
+* Type: String
+* Description: Correlation ID of event that triggered this event to be
+  created (if any). If not triggered in response to an event, a unique ID.
+  Can be specified by the producer if event not produced in response to an
+  event. Enables traceability through systems.
+* Examples:
+  * Event A generated with unique correlation-id. Event handler receives event
+    and generates a new event in response, Event B. Event B copies the
+    correlation-id of Event A so that the chain of events can be properly
+    correlated.
+* Constraints:
+  * Optional
+
+### causation-id
```

Comment:
@duglin Could we move this to the `Context Attributes Backlog`.  This was suggested in the Google doc by a contributor, but wasn't agreed to by anyone else.  We need to discuss and review new properties before adding them in.

Reply:
No idea why this comment didn't go away when I moved the entry, but I did move it as you can see.